srcpkgs/font-vazir-code/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'font-vazir-code'
+pkgname=font-vazir-code
+version=1.1.2
+revision=1
+create_wrksrc=yes
+depends="font-util"
+short_desc="Persian (farsi) monospaced font"
+maintainer="mkf <makefile@riseup.net>"
+license="custom:Public Domain"
```

Comment:
So, I'm not totally sure this is the license... According to https://github.com/rastikerdar/vazir-code-font/blob/master/LICENSE , I think we should include the bitstream license as well. Btw, you can just use "Public Domain", no need to make it custom.

Reply:
seems no.